td-agent/msi/Dockerfile

```diff
@@ -26,8 +26,8 @@ RUN @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -I
 RUN \
   choco install -y git wixtoolset 7zip & \
   choco install -y cmake --installargs 'ADD_CMAKE_TO_PATH=System' && \
-  choco install -y msys2 --params /NoUpdate --version=20220904.0.0 && \
-  choco install ruby -y --version=3.0.4.1 && \
+  choco install -y msys2 --params /NoUpdate --version=20221216.0.0 && \
```

Comment:
The latest msys2 version is 20230127.0.0. Shouldn't we use this or staying the previous one?

Reply:
With 20230127.0.0, it seems that .msi could be built successfully.

Reply:
Thanks, I didn't notice it.
I've updated it.